qlib/SqlUtil.qm

```diff
@@ -8476,8 +8475,8 @@ bool b = table.empty();
 
             # check options
             hash copt = getColumnOptions();
+            opt = opt{getTableCreationOptions().keys() + copt.keys()};
```

Comment:
please use a temporary variable here for the output of `getTableCreationOptions()` so that it's only called once (it's called again in the line below)